core/dom/range.js

```diff
@@ -2818,6 +2818,107 @@ CKEDITOR.dom.range = function( root ) {
 			reference.remove();
 		},
 
+		/**
+		 * Returns an array of {@link CKEDITOR.dom.rect} elements that are represented as rectangles which are covered by ranges.
+		 * For each DOM element within the selection range there is only one rectangle.
+		 * Rectangles represent the area of the screen occupied by the elements contained within the range.
+		 *
+		 * In the following example:
+		 *
+		 *        <p><span>first { span</span><span> second text</span></p>
+		 *        <p><span>very long } span</span></p>
+		 *
+		 * Brackets represent the beginning and the end of the selection.
+		 *
+		 * Returned rectangles would be represented by areas like below:
+		 *         text [ text ] [ text text ]
+		 *        [ text text ] text
+		 *
+		 * Where each pair of brackets represents one rectangle.
+		 *
+		 * In Internet Explorer 8 this method will return an array containing only one rectangle which would start in the top left hand corner of the selection and end in the bottom right hand corner.
+		 * The returned output represented by the scheme would be:
+		 *             ┌   ┐
+		 *        first span second span
+		 *        very long span
+		 *             └   ┘
+		 *
+		 * @since 4.10.0
+		 * @returns {Array} Returns an array of {@link CKEDITOR.dom.rect}
```

Comment:
This is not an optimal way of marking returned array. It should be marked as `@returns {Type[]}`.